portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -13,31 +13,31 @@
   public String email;
   public String key;
 
-  public static Map<String, Long> nLargestCommenters(Map<String, Long> hashMap, int n) {
-    Long value;
-    List<String> strings = new ArrayList<>(n); 
+  public static Map<String, Long> nLargestCommenters(Map<String, Long> accountIdToCommentCountMap, int n) {
+    Long commentCounts;
```

Comment:
Change from "commentCounts" to "commentCount".

Reply:
Changed this, thank you.